storybook/src/components/Button/Button.stories.tsx

```diff
@@ -12,14 +11,16 @@ const meta = {
   title: 'Components/Buttons/Button',
   component: Button,
   args: {
-    children: 'Button label',
+    children: 'Actietekst',
```

Comment:
‘Button label’ may not be the best (for one, because it’s more English than Dutch), but ‘Actietekst’ isn’t realistic content either.

Let’s use ‘Versturen’ as the default label for a primary button and ‘Annuleren’ for a secondary button. Not sure about the tertiary, though.

Reply:
I've opted for "Terug" for tertiary, I remember seeing it used for this purpose. I've also considered "Lees meer" like NLDS uses in button stories.  Buttons are getting quite short like this though.